.github/workflows/node-4+.yml

```diff
@@ -43,24 +44,46 @@ jobs:
             babel-eslint: 10
           - node-version: 4
             babel-eslint: 9
+          - node-version: 15
+            eslint: '^8.0.0-0'
+          - node-version: 13
+            eslint: '^8.0.0-0'
+          - node-version: 11
+            eslint: '^8.0.0-0'
+          - node-version: 11
+            eslint: 7
```

Comment:
why does this need to be excluded?

Reply:
It's not officially supported
https://github.com/eslint/eslint/blob/83cc8a6823d7729297b6814a644cca770cb99dbb/package.json#L143-L145

Reply:
Sure, but it works, so it still seems useful to test on it.